src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -71,41 +72,63 @@ protected function get_event( $event = null ) {
 	/**
 	 * Render the organizer chooser section for the events meta box
 	 *
+	 * @since ??
 	 */
 	public function render() {
 		$this->render_dropdowns();
 		$this->render_add_post_button();
 
 		/**
-		 * Make this Template filterable, used for Community Facing templates
+		 * Make this Template filterable, used for Community Facing templates.
+		 *
+		 * @since ??
 		 *
-		 * @var string $file_path
+		 * @param string $file_path
 		 */
 		include apply_filters( 'tribe_events_multiple_linked_post_template', $this->tribe->pluginPath . 'src/admin-views/linked-post-meta-box.php' );
 	}
 
 	/**
-	 * displays the saved organizer dropdown in the event metabox
-	 * Used to be a PRO only feature, but as of 3.0, it is part of Core.
+	 * Displays the saved linked post dropdown in the event metabox.
 	 *
+	 * @since 3.0
+	 * @since ?? Genericized to work for all linked posts, not just organizers like it was originally.
```

Comment:
The `??` here can be `TBD` since you're remark is about the change you implemented in this release cycle.